src/style.css

```
@@ -0,0 +1,110 @@
+body {
+  margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  margin: 0;
+  padding: 30px 14px 14px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 5px 45px;
```

```suggestion
  margin: 0 auto 45px;
```
just one more option